### PR TITLE
add paragraph about translation maintenance

### LINKED ISSUE #784: CONTRIBUTING.md and translation
- I suggest we move info in the README on contributing to a CONTRIBUTING.md file so it appears when creating a pull request
- I also would suggest some info on translation workflow and expectations and links to relevant documentation.  I had a small patch (#783), and had to dig around to find:
    - https://ropensci.org/blog/2023/09/26/how-to-translate-a-hugo-blog-post-with-babeldown/
    - https://translationguide.ropensci.org/howtoreview.html
- I don't believe any of this has guidance yet for what to do for PRs that are small changes to text within a larger body - should one update only that section or the while file?

### PR DESCRIPTION
**Pull request by maelle**
Fix #784

### CODE REVIEW

**Comment by noamross** [comment]
Thanks! I made a few tweaks.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -34,13 +34,16 @@ Then use `babelquarto::render_book()` and the book will be generated in the `_bo
...
 
-- a DeepL API key you can use for the dev guide, you can use [`babeldown::deepl_update()`](https://docs.ropensci.org/babeldown/reference/deepl_update.html) (see this [blog post](https://ropensci.org/blog/2024/01/16/deepl-update-babeldown/)), once per translation. Or, if there's only one sentence to be changed and you can see identify it in 
+- For your own translations, follow the guidance in the [rOpenSci Localization and Translation Guidelines](https://translationguide.ropensci.org)
 
-- no DeepL API key, please tag `@ropensci/dev-guide`.
+- To generate automated translations, you can use the {babeldown} package, wheich requires an API key from [DeepL](https://www.deepl.com). [`babeldown::deepl_update()`](https://docs.ropensci.org/babeldown/reference/deepl_update.html) generates translations to edited sections of the `.Rmd` (see this [blog post](https://ropensci.org/blog/2024/01/16/deepl-update-babeldown/)), and should be used once per translated langauge. 
 
-Then, the PR needs a reviewer for each language.
+- For tiny changes, (one sentence or so), tag the `@ropensci/dev-guide` team in your PR to request translations for your changes
```
Comment (maelle) [inline .github/CONTRIBUTING.md:44]: @noamross but we want editors to also tag us for bigger changes, as they have a) no API key b) maybe no more time to spend on their PR?

Comment (noamross) [reply .github/CONTRIBUTING.md]: 👍 Reworded

Comment (maelle) [reply .github/CONTRIBUTING.md]: thank you!!

**Pull request merged**